modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14268,6 +14268,7 @@ the-address-of-the-email-recipient=The address of the email recipient
 the-address-of-the-email-sender=The address of the email sender
 the-api-key-is-invalid=The API key is invalid.
 the-application-cannot-include-itself=The application cannot include itself.
```

Comment:
@yuhai I think you can directly change the message of this property?

Reply:
@dantewang, ok, I thought the property may be used by custom code.